apps/store/.env.example

```
@@ -7,6 +7,7 @@ VITE_IMAGEKIT_PUBLIC_KEY=
 VITE_IMAGEKIT_URL_ENDPOINT=
 VITE_POSTHOG_HOST=
 VITE_POSTHOG_KEY=
+VITE_FALAI_CI_PATTERN_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FALAI_CI_PATTERN_ENABLED key should go before the VITE_GA_ID key

(UnorderedKey)